trade-finance-manager-ui/server/controllers/case/amendments/amendCoverEndDate.test.js

```diff
@@ -19,14 +20,18 @@ const user = {
   email: 'test@localhost',
 };
 
-const facility = {
-  facilitySnapshot: { dates: { coverEndDate: '20 Oct 2020' } },
+const gefFacility = {
+  facilitySnapshot: { dates: { coverEndDate: '20 Oct 2020' }, type: FACILITY_TYPE.CASH },
```

Comment:
can this date be declared as a const as used more than once

Reply:
Refactored so not repeated